text/nodejs/0013-install-bom.md

```diff
@@ -61,36 +61,70 @@ the following fields for each module:
 All of these are fields that we aim to support per [the Paketo BOM
 RFC](https://github.com/sophiewigmore/rfcs/blob/8b1e8c9ed6201313f47c5897223cbffb265e96ed/text/0028-bill-of-materials.md)
 
-### Buildpack Additions
+In order to support this tool in both the online and offline cases, we will
+pre-compile the tool with it's dependencies, and provide it as a dependency
+hosted on the [dep-server](https://github.com/paketo-buildpacks/dep-server).
+
+### Language Family Additions
 
 Using the CycloneDX tool to generate the BOM will involve some changes to the
-NPM Install and Yarn Install Buildpacks.
+Node.JS language family buildpack order groups.
+
+* A single BOM generation buildpack will be added. It's primary functions are:
+  * perform the BOM dependency tool retrieval from the dep-server,
+  * execute BOM generation with the tool (`cyclonedx-bom -o bom.json`), and
+  * contribute the BOM to the layer metadata.
```

Comment:
Nitpick: You are not writing to the "layer metadata" right? You're gonna write to the {build,launch}.toml? (build/launch environment metadata - I don't know the technical term?) I think of layer metadata as `<layers>/<layer>.toml`. I think spelling it out would be clearer.

Reply:
See https://github.com/paketo-buildpacks/rfcs/pull/98/commits/7f6e30ec1eedcbb1221f9472bb97fa3464887a54